web/src/click_handlers.js

```diff
@@ -924,4 +924,11 @@ export function initialize() {
     $(".settings-header.mobile .fa-chevron-left").on("click", () => {
         settings_panel_menu.mobile_deactivate_section();
     });
+    // Workaround for issue #26941, which allows for element outside settings overlay
+    // to be focusable with the tab key
+    // https://github.com/zulip/zulip/issues/26941
+
+    $("#settings_end_focus").on("focus", () => {
+        document.querySelector("#settings_page").focus();
```

Comment:
Since we're using jQuery, probably best to stick with that (even though I am personally fonder of `document.querySelctor`).

Reply:
Alright. I've changed this to JQuery too.